src/KM_Controls.pas

```diff
@@ -105,22 +103,19 @@   TKMControl = class
     fOnClickRight: TPointEvent;
     fOnDoubleClick: TNotifyEvent;
     fOnMouseWheel: TNotifyEventMW;
-    fOnFocus: TNotifyEventFocus;
-    fOnControlMouseDown: TNotifyEventShift;
-    fOnControlMouseUp: TNotifyEventShift;

```

Comment:
8.Removed unnessesery events from TKMControl fOnControlMouseDown/Up. Must use ControlMouseDown/Up methods hierarhy instead.
This